deploy/fake-gpu-operator/values.yaml

```diff
@@ -60,11 +60,16 @@ migFaker:
     repository: gcr.io/run-ai-staging/fake-gpu-operator/mig-faker
     tag: 0.0.1
 
-initialTopology:
-  config:
-    node-autofill:
-      gpu-count: 2
-      gpu-product: Tesla-K80
-      gpu-memory: 11441
-  mig-strategy: mixed
-  nodes: {}
\ No newline at end of file
+topology:
+  # nodePools is a map of node pool name to node pool configuration.
+  # Nodes are assigned to node pools based on the node pool label's value (key is configurable via nodePoolLabelKey).
+  # 
+  # For example, nodes that have the label "run.ai/simulated-gpu-node-pool: default"
+  # will be assigned to the "default" node pool.
+  nodePools:
+    default:
+      gpuProduct: Tesla-K80
+      gpuCount: 2
+      gpuMemory: 11441
+  nodePoolLabelKey: run.ai/simulated-gpu-node-pool
+  mig-strategy: mixed
```

Comment:
why is it `mig-strategy` vs `migStrategy`?

Reply:
Oh thanks, my mistake